doc/src/build/zk_login.md

```diff
@@ -2,23 +2,24 @@
+zkLogin is a Sui primitive that let users send transactions from a Sui address using an OAuth credential, without publicly linking the two.
 
-zkLogin eliminates the need for users to handle private keys or recall mnemonics/passwords in wallets. By guiding users through the OAuth flow, an zkLogin address is generated, enabling any associated objects to execute transactions on-chain.
+zkLogin is designed with the following goals in mind:
 
-zkLogin is designed with the following principles in mind:
+1. **Simple Onboarding**: zkLogin allows users transact on Sui using the familiar OAuth login flow, eliminating the friction of handling cryptographic keys or remembering mnemonics.
 
-1. **Simple Onboarding**: With the familiar OAuth login flow, zkLogin enables users to easily create a Sui wallet and engage with Sui—eliminating the need for cryptographic key management.
+1. **Fully Self-Custodial**: A zkLogin transaction requires user approval via the standard OAuth login process--the OAuth provider cannot transact on the user's behalf.
 
-1. **Fully Self-Custodial**: A zkLogin transaction requires user approval within the OAuth login process, preventing unilateral fund movement from a zkLogin wallet—even by the OAuth provider.
+1. **Secure**: zkLogin is a two-factor authentication scheme: sending a transaction requires both a credential from a recent OAuth login and a password not managed by the OAuth provider. An attacker who compromises an OAuth account cannot transact from the user's corresponding Sui address unless they separately compromise the password.
 
-1. **Privacy-Focused**: Zero knowledge proofs prevent third parties from linking Sui addresses and OAuth identifiers.
+1. **Private**: Zero knowledge proofs prevent third parties from linking a Sui address with its corresponding OAuth identifier.
 
-1. **Natively Supported**: zkLogin is a protocol-level signature scheme, benefiting from [crypto agility](https://docs.sui.io/devnet/learn/cryptography/sui-signatures). Unlike smart contract-based alternatives, it will smoothly integrate with sponsored transactions and MultiSig.
+1. **Optional Verified Identity**: However, a user can *opt in* to verify the OAuth identifier that was used to derive a particular Sui address. This serves as the foundation for a verifiable on-chain identity layer.
 
-1. **Foundation for the Identity Layer**: In the future, zkLogin can serve as an opt-in identity layer for users on-chain.
+1. **Accessible**: zkLogin is one of several native Sui signature schemes thanks to Sui's [crypto agility](https://docs.sui.io/devnet/learn/cryptography/sui-signatures). It smoothly integrates with other powerful Sui primitives like sponsored transactions and multisig.
 
-Are you a builder who wants to integrate with zkLogin into your wallet or application? Dive into our [Integration guide](#integration-guide).
+
```

Comment:
@kchalkias, I'd like to add one more point called something like "Built with rigor" that will let us link to the proofs, audits, ceremony. The goal of this is to make the reader aware of the expertise and care that went into creating this. Could you help me suggest something?

Reply:
This is what we sent to ceremony participants to ensure security, audit and transparency.

If you are interested in what happens in the background during the ceremony, see https://sui.io/zklogin for more and FAQ.

Also, please feel free to review the open-source code repositories and documentation:

- Repo links:
    - Circuit: https://github.com/sui-foundation/zklogin-circuit
    - Ceremony: https://github.com/sui-foundation/zk-ceremony-client
- Doc links:
    - Developer docs: https://docs.sui.io/build/zk_login
    - Audit report from zkSecurity: https://github.com/sui-foundation/security-audits/blob/main/zksecurity_zklogin-circuits.pdf

Reply:
TL;DR:
- Credibility / Security: We allowed two software stacks for ZK ceremony contributions, there were more than 100 participants, many of them notable for their contribution in the general tech, security and blockchain community.
- Transparency: All related code has been open-sourced.
- Security: Both the ceremony and zklogin related code has been audited by at least two auditors.
- Innovation + Rigorousness: We have multiple unit tests, crosschecked and contributed in multiple open source projects related to Groth16 stack, in order to ensure compatibility and consistency.